modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.js

```diff
@@ -151,6 +151,16 @@ const TypeActivityKey = {
 	MISCELLANEOUS_MARKETING: 'prmtact004',
 };
 
+const ActivityStatus = {
+	active: 'active',
+	expired: 'expired',
+};
+
+const activityStatusClassName = {
+	[ActivityStatus.active]: 'label label-tonal-success ml-2',
+	[ActivityStatus.expired]: 'label label-tonal-danger ml-2',
+};
```

Comment:
Please try to substitute these enum properties with the Uppercase versions.

Reply:
OK!